packages/discord.js/src/structures/Poll.js

```diff
@@ -82,15 +111,55 @@ class Poll extends Base {
     } else {
       this.resultsFinalized ??= false;
     }
+
+    if (data.question) {
+      this.question = {
+        text: data.question.text,
+      };
+    }
+
+    if (data.answers) {
+      for (const answer of data.answers) {
+        const existing = this.answers.get(answer.answer_id);
+        if (existing) {
+          existing._patch(answer);
+        } else {
+          this.answers.set(answer.answer_id, new PollAnswer(this.client, answer, this));
+        }
+      }
+    }
+
+    if (data.partial) {
+      Object.defineProperty(this, '_partial', { value: data.partial });
+    }
```

Comment:
I believe this `_partial` property could just be removed. We should be able to keep only the `typeof` checks in the `partial` getter, which is also what's done in the rest of the entities that have partials